tanzu-java-web-app/accelerator.yaml

```diff
@@ -9,6 +9,7 @@ accelerator:
   - maven
   - web
   - tanzu
+  - devcontainer
```

Comment:
do we need to include the fragment in other java accelerators?

Reply:
That will require careful refactoring and consideration about each individual accelerator. In particular there is a 'problem' here that not all accelerators use the same name for `buildTool`. So I think that's this kind of refactoring is out of scope for this PR.

My goals for this PR where:
- create a devcontainer fragment
- use it at least once to prove it can be used
- ensure the most 'basic' sample `tanzu-java-web-app` gets devcontainer support.